components/Movie.js

```diff
@@ -29,13 +29,34 @@ const Id = styled.p`
 `;
 
 const Movie = ({ title, description, releaseYear, id }) => {
+  const [showDetails, setShowDetails] = useState(false);
+
+  const handleMovieClick = () => {
+    setShowDetails(true);
+  };
+
+  const handleCloseModal = () => {
+    setShowDetails(false);
```

Comment:
also here, better use the way of function NameOfFunction() {...} instead of an anonymous function :)